### DISCUSSION TITLE
Welcome to svelte-french-toast Discussions!

### DISCUSSION THREAD

**Discussion by kbrgl** [kbrgl/svelte-french-toast#41] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AkashskyX** [comment]
Im unable to pass props into the toast component 

`Argument of type '{ props: { status_code: any; error_message: any; }; style: string; position: "bottom-right"; }' is not assignable to parameter of type 'Partial<Pick<Toast, "id" | "icon" | "duration" | "ariaProps" | "className" | "style" | "position" | "iconTheme">>'.
  Object literal may only specify known properties, and 'props' does not exist in type 'Partial<Pick<Toast, "id" | "icon" | "duration" | "ariaProps" | "className" | "style" | "position" | "iconTheme">>'.js(2345)
(property) props: {
    status_code: any;
    error_message: any;
}`

this is the code 


function show__error_toast() {
        toast(Error, {
           
            props: {
        status_code: status_code,
        error_message: error_message
    },
            style: "background-color: transparent; box-shadow: none; margin-right:70px; padding: 0px;",
            position: "bottom-right",
            
        });
    }

**Reply by mrgk21** [reply]
This is merged in the master branch, but not included in the release.
 @kbrgl when are you guys planning the next release?
